google.json

```diff
@@ -457,7 +642,7 @@
     {
         "id": "com.google.android.apps.photos",
         "label": "Google Photos",
-        "description": "Allows Google to scan and catalog all your photos so that it knows you and your relations more than you do.",
+        "description": "Allows Google to scan and catalog all your photos so that it knows you and your relations more than you do.\nGood alternative for this can be also: Simple Gallery Pro based on AOSP gallery.",
```

Comment:
We don't suggest stuffs in the description. There's a dedicated section for it (which is already filled).